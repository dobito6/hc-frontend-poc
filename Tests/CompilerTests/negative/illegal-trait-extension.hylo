//! no-std

extension <T> U : P {}
extension A.B : P {}
extension f@let : P {}
