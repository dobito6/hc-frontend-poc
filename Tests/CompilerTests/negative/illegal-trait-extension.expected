negative/illegal-trait-extension.hylo:3.17-18: error: 'unexpected context bound'
extension <T> U : P {}
                ^
negative/illegal-trait-extension.hylo:4.15-16: error: 'unexpected context bound'
extension A.B : P {}
              ^
negative/illegal-trait-extension.hylo:5.17-18: error: 'unexpected context bound'
extension f@let : P {}
                ^
